### PR TITLE
fix: set description for selected items in milestone modal

### PR DESCRIPTION
**Pull request by adam-strzelec**
## Description

set description for selected items in milestone modal

## Testing

- Create staged payments action
- Go thru staged payments flow
- Open milestone modal and select one of decision method

## Diffs

**New stuff** ✨

Screenshot 2024-10-30 at 11 39 01

Screenshot 2024-10-30 at 11 39 14

**Changes** 🏗

Change translations for milestone modal

## TODO

Add translations for `Multi-sig` once they will be available

Resolves https://github.com/JoinColony/colonyCDapp/issues/3341

### CODE REVIEW

File: src/i18n/en.json [hunk 1]
```diff
@@ -1844,6 +1844,9 @@
     "fundingModal.note": "Note: ",
     "fundingModal.permissionsDescription": "This will use your {permission} permissions to immediately make the decision and fund the payment.",
     "fundingModal.reputationDescription": "This will start the lazy consensus process using reputation to make the decision for funding the payment.",
+    "milestoneModal.permissionsDescription": "This will use your {permission} permissions to immediately make the selected milestone payments.",
+    "milestoneModal.reputationDescription": "This will start the lazy consensus process using reputation to make the decision, for making the selected milestone payments.",
+    "milestoneModal.paymentCreatorDescription": "This will use your temporary permissions as the payment creator to immediately make the selected milestone payments.",
```
Comment (mmioana) [inline src/i18n/en.json:1849]: Can we also add the copy needed for `Multi-sig`?

File: src/components/v5/common/CompletedAction/partials/PaymentBuilder/partials/StagedPaymentStep/partials/MilestoneReleaseModal/consts.ts
Comment (mmioana) [inline src/components/v5/common/CompletedAction/partials/PaymentBuilder/partials/StagedPaymentStep/partials/MilestoneReleaseModal/consts.ts:1]: Can we please add also the `DecisionMethod.MultiSig` and comment it our for the moment?

**Review by mmioana — CHANGES_REQUESTED** [review]
All good here @adam-strzelec 💯 

The correct copy text is shown in the `Payment` modal for the selected decision method

`Permissions`
Screenshot 2024-10-31 at 11 23 51
`Payment creator`
Screenshot 2024-10-31 at 11 23 57
`Reputation`
Screenshot 2024-10-31 at 11 24 27

I'm requesting changes only because it would be nice to add already the needed copy and code for `Multi-Sig`, even if we comment it out for the moment

Also a side-note, it would be good to add in the testing steps that the `Staged payment` extension needs to be installed 💯

File: src/components/v5/common/CompletedAction/partials/PaymentBuilder/partials/StagedPaymentStep/partials/MilestoneReleaseModal/consts.ts [hunk 2]
```diff
@@ -8,18 +8,22 @@ export const getMilestoneReleaseDecisionMethodDescriptions = (
...
   [DecisionMethod.Reputation]: formatText({
-    id: 'fundingModal.reputationDescription',
+    id: 'milestoneModal.reputationDescription',
   }),
   [DecisionMethod.PaymentCreator]: formatText({
-    id: 'releaseModal.paymentCreatorDescription',
+    id: 'milestoneModal.paymentCreatorDescription',
   }),
+  // @TODO: Add multi-sig
+  // [DecisionMethod.PaymentCreator]: formatText({
```
Comment (mmioana) [inline src/components/v5/common/CompletedAction/partials/PaymentBuilder/partials/StagedPaymentStep/partials/MilestoneReleaseModal/consts.ts:24]: Here we should use `DecisionMethod.MultiSig` - but either way as it's disabled for the moment you can leave it as it is

**Review by mmioana — APPROVED** [review]
Thank you for adding these changes @adam-strzelec 💯 

The copy is showing up correctly for all decision methods
Screenshot 2024-11-06 at 11 07 33
Screenshot 2024-11-06 at 11 07 38
Screenshot 2024-11-06 at 11 07 42
Screenshot 2024-11-06 at 11 07 46

**Review by bassgeta — APPROVED** [review]
Nice and simple :ok_hand: 
image
image

**Review by Nortsova — APPROVED** [review]
Great work on this 👏 Thank you 

image
image
image
image

**Pull request merged**